server/src/entities/asset-face.entity.ts

```diff
@@ -37,6 +38,9 @@ export class AssetFaceEntity {
   @Column({ default: 0, type: 'int' })
   boundingBoxY2!: number;
 
+  @Column({ default: SourceType.MACHINE_LEARNING, type: 'enum', enum: SourceType })
+  sourceType!: string;
```

Comment:
This column should be non-nullable.

Reply:
Fixed by feea44af1028b988545fc712798e7c6a6fcca661